tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/Package/PackageCheckTool.cs

```diff
@@ -582,7 +582,7 @@ private static PackageCheckResponse CreateUnsupportedLanguageResponse(string pac
 
         private static bool IsNotImplemented(PackageCheckResponse response)
         {
-            return response.ResponseError != null && response.ResponseError.Contains("not implemented", StringComparison.OrdinalIgnoreCase);
+            return response.ResponseError != null && response.ResponseError.Contains("This is not an applicable operation for this language", StringComparison.OrdinalIgnoreCase);
```

Comment:
Do we really want this to be the generic message? Shouldn't we have a Not Applicable and a Not Implemented? Perhaps given this is a user message a Not Implemented isn't very helpful.

Reply:
I agree with not returning this message to the user but should we just make them no-op in the default case so there isn't any real error.